README.md

```diff
@@ -247,12 +248,16 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
    * [Gabriel Pato](https://www.youtube.com/channel/UC70YG2WHVxlOJRng4v-CIFQ)
    * [Linux Tips](https://www.youtube.com/user/linuxtipscanal)
 
+- Spanish
+  - [Shaylin Ortiz (ROOT OR DEATH](https://www.youtube.com/channel/UCZPiAjdoHhVNU7h9xnanGkw) Aws, Docker.
```

Comment:
Tbh, I won't feel comfortable including this YouTube channel (for now at least) as it still a new one so we cannot evaluate the quality of their content. I strongly believe that you can add other well known YouTube channels that produce a high-quality content